tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+# the integer IDs to experimental conditions or events. In this dataset, the
+# mapping looks like this:
 #
-# You can get indices corresponding to the good channels,
-# excluding ``raw.info['bads']``:
-
-picks = mne.pick_types(raw.info, meg=True, eeg=True, eog=True, stim=False,
-                       exclude='bads')
-
-##############################################################################
-# Alternatively one can restrict to magnetometers or gradiometers with:
-
-mag_picks = mne.pick_types(raw.info, meg='mag', eog=True, exclude='bads')
-grad_picks = mne.pick_types(raw.info, meg='grad', eog=True, exclude='bads')
-
-##############################################################################
-# Define the baseline period:
-
-baseline = (None, 0)  # means from the first instant to t = 0
-
-##############################################################################
-# Define peak-to-peak rejection parameters for gradiometers, magnetometers
-# and EOG:
-
-reject = dict(grad=4000e-13, mag=4e-12, eog=150e-6)
-
-##############################################################################
-# Read epochs:
-
-epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
-                    picks=('meg', 'eeg', 'eog'),  # or could pass "picks" here
-                    baseline=baseline, preload=False, reject=reject)
-print(epochs)
-
-##############################################################################
-# Get single epochs for one condition:
-
-epochs_data = epochs['aud_l'].get_data()
-print(epochs_data.shape)
-
-##############################################################################
-# epochs_data is a 3D array of dimension (55 epochs, 365 channels, 106 time
-# instants).
+# +----------+----------------------------------------------------------+
+# | Event ID | Condition                                                |
+# +==========+==========================================================+
+# | 1        | auditory stimulus (tone) to the left ear                 |
+# +----------+----------------------------------------------------------+
+# | 2        | auditory stimulus (tone) to the right ear                |
+# +----------+----------------------------------------------------------+
+# | 3        | visual stimulus (checkerboard) to the left visual field  |
+# +----------+----------------------------------------------------------+
+# | 4        | visual stimulus (checkerboard) to the right visual field |
+# +----------+----------------------------------------------------------+
+# | 5        | smiley face (catch trial)                                |
+# +----------+----------------------------------------------------------+
+# | 32       | subject button press                                     |
+# +----------+----------------------------------------------------------+
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
```

Comment:
There are two types of dictionary constructions that have been used in this tutorial. To avoid confusion for a beginner, I would suggest sticking to one -- either dict() or {}

Reply:
we can't use `dict()` here because of the `/` in the keys, but we have a general guidance to use `dict()` where possible because it's more transparent...  Is consistent use of `{'key': val}` throughout the tutorial (and every tutorial? or at least every one with an event dict) better even though it's less transparent than `dict()`?

Reply:
I think using `{}` where needed is fine. There are times it's nicer and/or necessary to use this syntax if you want to do everything in the constructor.